src/firebase/snap/Snap.ts

```diff
@@ -0,0 +1,7 @@
+/* Snap Structure */
+export default interface Snap {
+    to : [string];  /* Recipient UIDs */
+    from: string;   /* Sender UID */
+    body: string;   /* Main Message */
+    timestamp: Date;   /* Date/time of Submission*/
```

Comment:
Does Firestore support `Date` types natively?

Reply:
it all seems to work okay when we put them in!